inji-web/src/__tests__/components/DataShare/DataShareFooter.test.tsx

```diff
@@ -0,0 +1,43 @@
+import {fireEvent, render, screen} from "@testing-library/react";
+import {DataShareFooter} from "../../../components/DataShare/DataShareFooter";
+
+describe("Testing Layout of the Expiry Footer", () => {
+
+    const successMockFn = jest.fn();
+    const cancelMockFn = jest.fn();
+    beforeEach(() => {
+        jest.mock("react-i18next", () => ({
+            useTranslation: () => ({
+                t: (key: string) => key,
+            }),
+        }));
+        render(<DataShareFooter success={"success"} onSuccess={successMockFn} cancel={"cancel"} onCancel={cancelMockFn} />);
+    } )
+
+    test("Test the presence of the Outer Container", ()=>{
+        const document = screen.getByTestId("DataShareFooter-Outer-Container");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Cancel Button", ()=>{
+        const document = screen.getByTestId("DataShareFooter-Cancel-Button");
+        expect(document).toBeInTheDocument();
+    })
+    test("Test the presence of the Success Button", ()=>{
+        const document = screen.getByTestId("DataShareFooter-Success-Button");
+        expect(document).toBeInTheDocument();
+    })
```

Comment:
single snapshot test to cover it

Reply:
Snapshot Test we will take it up as part of different card mentioned.